opm/ml/ml_tools/scaler_layers.py

```diff
@@ -0,0 +1,210 @@
+#  Copyright (c) 2024 NORCE
+#  Copyright (c) 2024 UiB
+
+#   This file is part of the Open Porous Media project (OPM).
+
+#   OPM is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+
+#   OPM is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+
+#   You should have received a copy of the GNU General Public License
+#   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Provide MinMax scaler layers for tensorflow.keras."""
+
+from __future__ import annotations
+
+from typing import Optional, Sequence
+
+import numpy as np
+import tensorflow as tf
+from numpy.typing import ArrayLike
+from tensorflow import keras
```

Comment:
When using external libraries I think it would be a good idea to supply a requirements.txt file to fix versions, so that a future user will be able to run these scripts.